utils/firebaseNotification.js

```diff
@@ -0,0 +1,74 @@
+import messaging from '@react-native-firebase/messaging'
+import { Alert } from 'react-native';
+
+
+export default class FirebaseNotification{
+
+async checkPermission(){
+    const authStatus = await messaging().requestPermission();
+    const enabled =
+      authStatus === messaging.AuthorizationStatus.AUTHORIZED ||
+      authStatus === messaging.AuthorizationStatus.PROVISIONAL;
```

Comment:
Instead of that try something like this

```js
const { AUTHORIZED, PROVISIONAL } = messaging.AuthorizationStatus;
const isAuthorized = authStatus === AUTHORIZED;
const isProvisional = authStatus === PROVISTIONAL;

const isEnabled = isAuthorized || isProvisional;
```

Reply:
Fixed